### PR TITLE
Activity new clients for current month docs

### PR DESCRIPTION
**Pull request by HridoyRoy**

### CODE REVIEW

File: website/content/docs/concepts/client-count/index.mdx [hunk 1]
```diff
@@ -253,3 +253,58 @@ For the UI to be able to use the client count feature, it needs `read` permissio
 
 For the UI to be able to modify the configuration settings, it additionally needs `update` permission to
 `sys/internal/counters/config`.
+
+## New Clients For Current Month
+
+As of Vault 1.12, the billing period for the activity log API can be specified to include the current month
```
Comment (HridoyRoy) [inline website/content/docs/concepts/client-count/index.mdx:259]: Notes from Vishal sync: we should add a disclaimer/small paragraph explaining the reason for the approximation (ie why it's necessary) without getting too technical, but explaining that it's a purely technical reason, a general computing limitation.

Comment (HridoyRoy) [reply website/content/docs/concepts/client-count/index.mdx:259]: Make sure to stress that this is done primarily to improve API performance and make the UI usable. Mention the problem name (count-distinct for multisets) and explain why this maps to that.

**Review by vishalnayak — APPROVED** [review]
Outside of the 1 pending comment, this is looking good to me. Great job!

**Review by finnstech — APPROVED** [review]
Looks good to me. Some of the sentences could be edited down for brevity/clarity if we do another pass.

File: website/content/docs/concepts/client-count/index.mdx [hunk 2]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:263]:
```suggestion
The billing period for the activity log API can be specified to include the current month 
```

File: website/content/docs/concepts/client-count/index.mdx [hunk 3]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
 `sys/internal/counters/config`.
+
+## New Clients For Current Month
+
+As of Vault 1.12, the billing period for the activity log API can be specified to include the current month 
+for the end date. For more information about how this can be done, please visit 
+[the internal counters API docs](/api-docs/system/internal-counters.mdx). 
+
+When the end date is the current month, the `new_clients` counts will be an approximation of the true 
+number of new clients for the month, not an exact value. Note that the `new_clients` counts for the rest
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:268]:
```suggestion
number of new clients for the month, and not an exact value. Note that the `new_clients` counts for the rest 
```

File: website/content/docs/concepts/client-count/index.mdx [hunk 4]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
+
+## New Clients For Current Month
+
+As of Vault 1.12, the billing period for the activity log API can be specified to include the current month 
+for the end date. For more information about how this can be done, please visit 
+[the internal counters API docs](/api-docs/system/internal-counters.mdx). 
+
+When the end date is the current month, the `new_clients` counts will be an approximation of the true 
+number of new clients for the month, not an exact value. Note that the `new_clients` counts for the rest 
+of the months will all be accurate.
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:269]:
```suggestion
of the months will be accurate. 
```

File: website/content/docs/concepts/client-count/index.mdx [hunk 5]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
+[the internal counters API docs](/api-docs/system/internal-counters.mdx). 
+
+When the end date is the current month, the `new_clients` counts will be an approximation of the true 
+number of new clients for the month, not an exact value. Note that the `new_clients` counts for the rest 
+of the months will all be accurate. 
+
+### Why An Approximation?
+
+The `new_clients` counts for the current month is an approximation primarily to improve API 
+performance and make the UI usable. In order to give an exact value for the current month's
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:274]:
```suggestion
performance and make the UI usable. To give an exact value for the current month's
```

File: website/content/docs/concepts/client-count/index.mdx [hunk 6]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
+new clients, client IDs need to be de-duplicated with the previous months' client IDs, 
+which is a time and i/o intensive process. 
+
+### Approximation Details And Accuracy Testing Results
+
+The `new_clients` approximation is calculated is by using a [hyperloglog algorithm](https://en.wikipedia.org/wiki/HyperLogLog)
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:280]:
```suggestion
The `new_clients` approximation is calculated by using a [hyperloglog algorithm](https://en.wikipedia.org/wiki/HyperLogLog)
```

File: website/content/docs/concepts/client-count/index.mdx [hunk 7]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
+to approximate the cardinality of the total number of clients within the billing period, and the cardinality
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:281]:
```suggestion
to approximate the cardinality of the total number of clients within the billing period. The cardinality 
```

File: website/content/docs/concepts/client-count/index.mdx [hunk 8]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
+The `new_clients` counts for the current month is an approximation primarily to improve API 
+performance and make the UI usable. In order to give an exact value for the current month's
+new clients, client IDs need to be de-duplicated with the previous months' client IDs, 
+which is a time and i/o intensive process. 
+
+### Approximation Details And Accuracy Testing Results
+
+The `new_clients` approximation is calculated is by using a [hyperloglog algorithm](https://en.wikipedia.org/wiki/HyperLogLog)
+to approximate the cardinality of the total number of clients within the billing period, and the cardinality 
+of the total number of clients within the billing period not including the current month. The returned value
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:282]:
```suggestion
of the total number of clients within the billing period does not the current month. The returned value 
```

File: website/content/docs/concepts/client-count/index.mdx [hunk 9]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
+
+### Approximation Details And Accuracy Testing Results
+
+The `new_clients` approximation is calculated is by using a [hyperloglog algorithm](https://en.wikipedia.org/wiki/HyperLogLog)
+to approximate the cardinality of the total number of clients within the billing period, and the cardinality 
+of the total number of clients within the billing period not including the current month. The returned value 
+is the difference between these two numbers.
+
+The hyperloglog library used for the cardinality estimate is [axiomhq](https://github.com/axiomhq/hyperloglog),
+with 14 registers and the use of sparse representations when applicable. Some accuracy estimates can be found
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:286]:
```suggestion
with fourteen registers and the use of sparse representations, when applicable. Some accurate estimates can be found 
```

File: website/content/docs/concepts/client-count/index.mdx [hunk 10]
```diff
@@ -257,3 +257,67 @@ For the UI to be able to use the client count feature, it needs `read` permissio
...
+with 14 registers and the use of sparse representations when applicable. Some accuracy estimates can be found 
+in this library's [README](https://github.com/axiomhq/hyperloglog#readme). These are accuracy results
+for the cardinality of the multiset, which is the total number of clients within a billing period. The accuracy 
+estimate for the number of new clients can be far lower, depending on the discrepancy between the number of 
+clients in the current month and the number of clients in the billing period. 
+
+If we call the number of clients for the current month C and the number of clients in the billing period B, we 
+have found that in general, if C << B, the approximation can be imprecise, and the further the difference between
+C and B grows, the more imprecise the approximation will be. Furthermore, the closer C is to 0, the more imprecise
+the approximation can be. Also, the more months in the billing period the less accurate the approximation can be.
```
Comment (taoism4504) [inline website/content/docs/concepts/client-count/index.mdx:295]:
```suggestion
the approximation can be. Also, the more months in the billing period, the less accurate the approximation can be. 
```

File: website/content/api-docs/system/internal-counters.mdx [hunk 11]
```diff
@@ -257,9 +259,69 @@ information, it will not be added to the API response.
 match the query parameters due to the month granularity of data or missing
 months in the requested time range.
 
+- Note that if the `end_date` specified is the current month (available as of 1.12),
```
Comment (taoism4504) [inline website/content/api-docs/system/internal-counters.mdx:262]: Don't think the release number needs to be called out.

**Review by taoism4504 — APPROVED** [review]
Some edits/suggestions. Please accept as you see fit. If any of my edits changed the intent of the sentence, please feel free to ignore the suggestion.

**Pull request merged**